<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaskedManInQtClass</class>
 <widget class="QDialog" name="MaskedManInQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaskedManInQt</string>
  </property>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QSlider" name="play_slider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>321</width>
     <height>19</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_openVideo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开视频</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>蒙面人智能识别系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>信息检测框</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="info_text">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="maskedmaninqt.qrc"/>
 </resources>
 <connections/>
</ui>
